kotsadm/web/src/components/apps/AppVersionHistory.jsx

```diff
@@ -149,43 +150,67 @@ class AppVersionHistory extends Component {
     return (
       <div>
         {diffSummary ?
-          (diffSummary.filesChanged > 0 ?
-            <div
-              className="DiffSummary u-cursor--pointer"
-              onClick={() => {
-                if (!downstream.gitops?.enabled) {
-                  this.setState({
-                    showDiffOverlay: true,
-                    firstSequence: version.parentSequence - 1,
-                    secondSequence: version.parentSequence
-                  });
-                }
-              }}
-            >
-              <span className="files">{diffSummary.filesChanged} files changed </span>
-              <span className="lines-added">+{diffSummary.linesAdded} </span>
-              <span className="lines-removed">-{diffSummary.linesRemoved}</span>
+          (diffSummary.filesChanged > 0 && yamlErrorsDetails ?
+            <div className="flex flex1 alignItems--center">
+              <div
+                className="DiffSummary u-cursor--pointer"
+                onClick={() => {
+                  if (!downstream.gitops?.enabled) {
+                    this.setState({
+                      showDiffOverlay: true,
+                      firstSequence: version.parentSequence - 1,
+                      secondSequence: version.parentSequence
+                    });
+                  }
+                }}
+              >
+                <span className="files">{diffSummary.filesChanged} files changed </span>
+                <span className="lines-added">+{diffSummary.linesAdded} </span>
+                <span className="lines-removed">-{diffSummary.linesRemoved}</span>
+              </div>
+              <div className="flex flex1 alignItems--center u-marginLeft--10">
```

Comment:
i feel like the `yamlErrorsDetails` can be used here instead of duplicating the whole div